src/gentropy/dataset/study_index.py

```diff
@@ -183,3 +183,14 @@ def has_summarystats(self: StudyIndex) -> Column:
             Column: True if the study has harmonized summary statistics.
         """
         return self.df.hasSumstats
+
+    def get_major_pop(self: StudyIndex) -> DataFrame:
+        """Extract major population from ldPopulationStructure rows with multiple ancestries.
+
+        Returns:
+            DataFrame: Columns studyId and the extracted major population from ldPopulationStructure.
+        """
+        return self.df.select("studyId", "ldPopulationStructure").withColumn(
+            "ldPopulationStructure",
+            f.array_max(f.col("ldPopulationStructure")).getItem("ldPopulation"),
+        )
```

Comment:
I would rather do something like this:
```suggestion
        return self.df.select(
            "studyId",
            f.array_max(f.col("ldPopulationStructure"))
            .getItem("ldPopulation")
            .alias("majorPopulation"),
        )
```
The name of the returned column should be different from what it was originally called. Also I would not use abbrevation in the function name. (`get_major_pop` -> `get_major_population`)